app/controllers/users/sessions_controller.rb

```diff
@@ -13,17 +13,10 @@ def respond_with(resource, _opts = {})
     end
 
     def respond_to_on_destroy
-      if current_user
-        render json: {
-          status: 200,
-          message: 'logged out successfully'
-        }, status: :ok
-      else
-        render json: {
-          status: 401,
-          message: "Couldn't find an active session."
-        }, status: :unauthorized
-      end
+      render json: {
+        status: 200,
```

Comment:
Why do you need to include the `status` in the json object? Wouldn't HTTP status 200 already be enough for the frontend to know that it is successful?